projects/plugins/jetpack/extensions/shared/test/block-fixtures.md

```diff
@@ -0,0 +1,101 @@
+# Block test fixtures
```

Comment:
Can we also add a section on how to run these tests?

Reply:
Done.